sdk/eventhub/ci.yml

```diff
@@ -41,6 +41,7 @@ stages:
 - template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: eventhub
+    ToxEnvParallel: ' '
```

Comment:
I assume this is just a temporary fix until eventhubs can run in parallel? If so I'm fine with this change but we should file an issue for reverting this. If not then I'd like to better understand why we cannot run in parallel.

Reply:
I'm fine with that. Pretty damn confident that it's not a CI issue, but need to unblock release.

Reply:
[Issue filed](https://github.com/Azure/azure-sdk-for-python/issues/9429)